waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -126,7 +124,10 @@ npmDepsForWasp spec =
             ("helmet", "^6.0.0"),
             ("patch-package", "^6.4.7"),
             ("uuid", "^9.0.0"),
-            ("lodash", "^4.17.21")
+            ("lodash", "^4.17.21"),
+            -- todo make optional
+            -- todo types for node
```

Comment:
```suggestion
            -- TODO: make optional
            -- TODO: types for node
```
because it is how we do it in the rest of the codebase + more editors paint it nicely.